src/main/java/io/xpring/ilp/DefaultIlpClient.java

```diff
@@ -67,39 +65,45 @@ public DefaultIlpClient(final ManagedChannel channel) {
     }
 
     @Override
-    public AccountBalance getBalance(final String accountId, final String bearerToken) throws XpringException {
+    public AccountBalance getBalance(final String accountId, final String accessToken) throws XpringException {
         GetBalanceRequest request = GetBalanceRequest.newBuilder()
           .setAccountId(accountId)
           .build();
 
         try {
             GetBalanceResponse response = this.balanceServiceStub
-              .withCallCredentials(IlpJwtCallCredentials.build(bearerToken))
+              .withCallCredentials(IlpCredentials.build(accessToken))
               .getBalance(request);
 
             // Convert protobuf response to AccountBalanceResponse
             return AccountBalance.from(response);
         } catch (StatusRuntimeException e) {
             throw new XpringException(String.format("Unable to get balance for account %s.  %s", accountId, e.getStatus()));
+        } catch (IllegalArgumentException e) {
```

Comment:
Don't use one letter variables. Call this `exception` or `illegalArgumentException`.

Reply:
This is pretty common in Java, but will change anyway